### ISSUE TITLE
Countdown Equipment that have been Cursed (and invisible) still accept dice

### ISSUE THREAD

**Issue by ThxSkeleton** [TerryCavanagh/diceydungeons.com#1210]
My Specific Scenario:

Warrior Episode 2:
- Got the Countdown charm that will roll 2 sixes
- Attempt to use the charm by dragging a dice onto it
- It gets cursed (as normal)
- Charm graphic disappears (as normal)
- I drag a dice to the spot where the dice slot would be (if the charm still existed)
- Unexpectedly, the "dice consumed" sound effect plays, and the dice disappears.

**Comment by bierjulius** [comment]
It might be coincidence but the same thing happened to me also with warrior on episode 2 and midnight charm+. Attached is a video of the bug.

[https://youtu.be/Xi1Bb3lqA4I](url)

**Comment by ThxSkeleton** [comment]
@bierjulius I'm guessing it's a general sort of bug but we2 has a lot of curses

**Comment by Apreche** [comment]
I wasn't able to get a video of the bug, but I experienced this today. I got a curse on a countdown equipment. Even though the equipment no longer appeared on the screen, it ate my dice when I dropped the in its former location. Even on the next turn the equipment did not reappear! I forget if it reappeared the turn after that or if I had to finish the fight. Either way, it was definitely back to normal when I was fighting the next enemy.
